1.4/Patches/Buildings.xml

```diff
@@ -360,8 +360,8 @@
 				<xpath>Defs/ThingDef[defName="CeramicTileWall"]</xpath>
 				<value>
 					<li Class="Celsius.ThingThermalProperties">
-						<heatCapacity>11</heatCapacity>
-						<insulation>3.5</insulation>
+						<heatCapacity>8</heatCapacity>
+						<insulation>1.5</insulation>
```

Comment:
Ceramic shouldnt be simular to brick, its quite a good insulator and has decent heat capacity